fastlane/Fastfile

```diff
@@ -408,6 +408,19 @@ end
       dark_mode: options[:mode].to_s.downcase == "dark"
     )
   end
+
+  lane :create_promo_screenshots do |options|
+    unless Fastlane::Helper::GitHelper.has_git_lfs then
```

Comment:
Nitpick. I prefer `unless condition then` to `if !condition then` or `if condition == false then`.

I've attended a talk on code readability making the point that `!` are sneaky and easily overlooked when reading code. I don't have a paper with a real study to back this up, but it does make sense to me. From that day, I've stayed away from `!something` whenever possible.